package.json

```diff
@@ -48,6 +48,8 @@
     "rimraf": "^2.5.2"
   },
   "dependencies": {
+    "airtable": "^0.5.2",
```

Comment:
This isn't being used, please remove.

Reply:
Oh shoot, this PR's off a branch, isn't it...
That's in place pending implementation as a Karma backend (soon).
If you want, we can revert and drop a branch with this deleted at ~that point in time :)

Reply:
You can also just throw up another commit removing the dependencies, less work for you.